### PR TITLE
Fix doc api oas yaml - remove un-implemented predicates in where

### PR DESCRIPTION
**Pull request by polandll**

### CODE REVIEW

File: modules/developers-guide/examples/graphql/schema-new.graphql [hunk 1]
```diff
@@ -0,0 +1,65 @@
+type Address @cql_entity(target: UDT) @cql_input {
```
Comment (dimas-b) [inline modules/developers-guide/examples/graphql/schema-new.graphql:1]: Hmmm .... do you mean to update GraphQL docs as well? This does not seem related to Docs API "where" clauses :thinking:

**Comment by polandll** [comment]
Made from data_fed branch rather than master.
